client/js/components/procedure/SegmentsList/SegmentsList.vue

```diff
@@ -498,9 +502,15 @@ export default {
         })
         .then(data => {
           this.isLoading = false
```

Comment:
```suggestion
        .finally(data => {
          this.isLoading = false
```

I think the loading state should be set to false in a finally block because if the request somehow fails, the loading state will not be set to false.